sunpy/net/vso/vso.py

```diff
@@ -207,148 +113,6 @@ def build_client(url=None, port_name=None, **kwargs):
     return client
 
 
-class QueryResponse(BaseQueryResponse):
```

Comment:
This should be deprecated instead of being removed

Reply:
This has just moved house into `vso/legacy_response.py`, I should probably import it back into here just to be safe.